### PR TITLE
chore(theia-dev container): adding libsecret and tig to theia dev container

### PR DESCRIPTION
**Pull request by sunix**
### What does this PR do?
adding libsecret and tig for theia-dev container to be used in theia and che-theia dogfooding devfiles
- libsecret is required now by Theia. And it is impossible to run yarn start without it.
- tig is a nice cli tool for git to be used by developers.

### Screenshot/screencast of this PR
N/A

### What issues does this PR fix or reference?
N/A

### How to test this PR?
build theia-dev container, push it to quay.io, and use it with theia devfile instead the default theia-dev image https://github.com/eclipse-theia/theia/blob/master/devfile.yaml#L15

running theia after having built it should work.

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [ ] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [ ] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [ ] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### Happy Path Channel

HAPPY_PATH_CHANNEL=stable

### CODE REVIEW

File: dockerfiles/theia-dev/docker/alpine/install-dependencies.dockerfile [hunk 1]
```diff
@@ -16,4 +16,8 @@ RUN apk add --update --no-cache \
     # synchronization tool
     rsync \
     # patch (required in che-theia to apply patches)
-    patch
+    patch \
+    # requirements to run theia with yarn start
+    libsecret \
+    # git tooling for developers
+    tig
```
Comment (benoitf) [inline dockerfiles/theia-dev/docker/alpine/install-dependencies.dockerfile:23]: I would not add this library if it's not added in the ubi8 image as well

Comment (sunix) [reply dockerfiles/theia-dev/docker/alpine/install-dependencies.dockerfile:23]: It's not available in the ubi repo's :/
Is someone using the ubi8 image ? does it make sense to maintain it ?

Comment (sunix) [reply dockerfiles/theia-dev/docker/alpine/install-dependencies.dockerfile]: @benoitf I have removed tig ... but it is a shame :/

**Review by benoitf — APPROVED** [review]

**Comment by nickboldt** [comment]
Note that this fix doesn't work for downstream on s390x and ppc64le. https://issues.redhat.com/browse/CRW-1995

**Pull request merged**